bitnami/redis/Chart.yaml

```diff
@@ -19,4 +19,10 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 12.3.3
+version: 12.3.4
+dependencies:
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 1.x.x
```

Comment:
We order the `Chart.yaml` file alphabetically, could you move this above `description`?